### ISSUE TITLE
Does it work with Toolbar items?

### ISSUE THREAD

**Issue by ParitoshBh** [sephiroth74/android-target-tooltip#24]
This is more of a question than an issue, so you can close it without even answering!

Anyhow, in documentation I couldn't find if this library works for showing tooltips for toolbar items.

**Comment by JungleTian** [comment]
:+1:

**Comment by jacobrs** [comment]
:+1:

**Comment by sephiroth74** [comment]
yes it works, as long as you're able to find the view below the view inside the tablayout
